packages/@ember/-internals/glimmer/tests/integration/components/input-curly-test.js

```diff
@@ -439,11 +439,11 @@ moduleFor(
             },
           },
         });
-      }, 'Passing actions to components as strings (like `{{input key-press="foo"}}`) is deprecated. Please use closure actions instead (`{{input key-press=(action "foo")}}`). (\'-top-level\' @ L1:C0) ');
+      }, /(Passing actions to components as strings \(like `{{input key-press="foo"}}`\) is deprecated\.|Passing actions to components as strings \(like `<Input @key-press="foo" \/>`\) is deprecated\.)/);
```

Comment:
To make this more readable let's restrict the `|` to the part that's actually different:
```suggestion
      }, /Passing actions to components as strings \(like `({{input key-press="foo"}}|<Input @key-press="foo" \/>)`\) is deprecated\./);
```

Reply:
(same for the rest)

Reply:
That definitely looks better. I updated all of them to that format.